seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1262,10 +1262,10 @@
       "challengeType": 0,
       "translations": {
         "de": {
-          "title": "Erstelle ein Textfeld",
+          "title": "Ein Textfeld erstellen",
           "description": [
             "Nun erstellen wir ein Web Formular.",
-            "Eingabefelder sind eine bewährte Möglichkeit um Daten von Usern zu erhalten.",
+            "Eingabefelder sind eine beliebte Möglichkeit um Daten von Usern zu erhalten.",
```

Comment:
this changes the meaning, "bewährte" means it is reliable and therefore should be used, while "beliebte" indicates a preference